integration_tests/suite/helpers/bus.py

```diff
@@ -54,8 +81,11 @@ def send_session_created_event(
                 },
                 'name': 'auth_session_created',
             },
-            routing_key=f'auth.sessions.{session_uuid}.created',
-            headers={'tenant_uuid': str(tenant_uuid)},
+            headers={
+                'name': 'auth_session_created',
+                'tenant_uuid': str(tenant_uuid),
+            },
+            routing_key='config.user.deleted',
```

Comment:
the `routing_key` got changed from `f'auth.sessions.{session_uuid}.created'` to `'config.user.deleted'`